eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -9,7 +9,9 @@ function CreateReleases($pkgList, $releaseApiUrl, $releaseSha) {
 
     $releaseNotes = ""
     if ($pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseContent -ne $null) {
-      $releaseNotes = $pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseContent
+      [string]$releaseNotesTitle = $pkgInfo.ReleaseNotes[$pkgInfo.PackageVersion].ReleaseTitle
```

Comment:
I'd suggest doing this in Extract-ReleaseNotes.ps1 script (or the helpers it calls). If all that is needed is the content for one version then return the string for the exact release notes in the Extract-ReleaseNotes script.

Reply:
Actually this script initially pulls our the entire collection of releaseNotes object. It passes that around and only pulled out the exact release note just before creating the release. I think its better leave both functions. Otherwise I will need to refactor other parts of the `artifact-metadata-parsing.ps1` script which seems unecessary.

Reply:
I would like to clean this script up to use not only the new change log stuff but also the new package metadata helpers. I suggest undoing the changes to this file for this PR and keep it calling the existing script and do this work as a separate change.

Reply:
I updated the script to used the new function I added that returns just string.